AdvancedSamples/TestApp/app/src/main/java/com/kaltura/kalturaplayertestapp/PlayerActivity.kt

```diff
@@ -1591,6 +1591,27 @@ class PlayerActivity : AppCompatActivity(), Observer {
             }
         }
 
+        player?.addListener(this, PhoenixAnalyticsEvent.bookmarkError) { event ->
+            val reportedEventName = event.type.name
+            if (PlayerEvent.Type.PLAYHEAD_UPDATED.name != reportedEventName) {
+                updateEventsLogsList("phoenix:\n$reportedEventName")
+            }
+        }
+
+        player?.addListener(this, PhoenixAnalyticsEvent.concurrencyError) { event ->
+            val reportedEventName = event.type.name
+            if (PlayerEvent.Type.PLAYHEAD_UPDATED.name != reportedEventName) {
+                updateEventsLogsList("phoenix:\n$reportedEventName")
+            }
+        }
+
+         player?.addListener(this, PhoenixAnalyticsEvent.error) { event ->
```

Comment:
@GouravSna  this is report you want to use here?

o/w what is PLAYHEAD_UPDATED is here?

Reply:
Have cleaned up this redundant condition from everywhere. it is not required anywhere, i think

Reply:
PTAL